server/src/test/java/io/crate/planner/operators/EquiJoinDetectorTest.java

```diff
@@ -123,13 +123,18 @@ public void test_equality_expression_followed_by_case_expression() {
     @Test
     public void test_equality_and_many_relations_in_boolean_join_condition_hash_join_not_possible() {
         // Nested EQ operator.
+        // failed before my fix not only because of = but also coincided that t1 got reset and it left t1 - t2 on top level
```

Comment:
Please rephrase this, `my fix` should be replaced maybe with link to PR.
I also don't understand the sentence: `reset` and `left t1 - t2 on top level`.

Reply:
Oops, that's "self documenting", a debugging artefact and has no value. I will remove it. Thanks for spotting!

Reply:
https://github.com/crate/crate/pull/16245/